Sprint-2/stretch/till.js

```diff
@@ -8,10 +8,15 @@ function totalTill(till) {
   let total = 0;
 
   for (const [coin, quantity] of Object.entries(till)) {
-    total += coin * quantity;
+    const coinValue = Number(coin.slice(0, -1));
```

Comment:
This works but please try another better way of getting the number value from the input string.

Please refer: https://www.freecodecamp.org/news/java-string-to-int-how-to-convert-a-string-to-an-integer/

Reply:
The link you've added is for Java, not JavaScript. I have taken the concept though and applied it. I've changed that line of code to `const coinValue = parseInt(coin, 10);`